crates/next_version/src/config.rs

```diff
@@ -0,0 +1,26 @@
+/// Configuration for version incrementation rules.
+#[derive(Debug)]
+pub struct NextVersionConfig {
+    /// Indicates whether uncontrolled minor version bumps are enabled.
+    /// When false, minor version increments are not performed and they
+    /// should be manually done to signal higher API stability to the user.
+    ///
+    /// Default value is `false`.
```

Comment:
```suggestion
    /// Default value is `true`.
```
I see that the default is true.

Reply:
This comment was wrong, updated.